### DISCUSSION TITLE
Slow performance in 3.2/3.3 compared to 2.8

### DISCUSSION THREAD

**Discussion by tompollok** [pelicanmapping/osgearth#2084] [Support Q&A]
Hello,

Im experiencing a significant slower performance with the latest osg versions (3.x) compared to the old version (2.x).

In 3.x, for loading geotiff image layers, im using now:

osg::ref_ptr<osgEarth::GDALImageLayer> layer = new osgEarth::GDALImageLayer();
layer->setURL(path.toStdString());
map->addLayer(layer);

For loading elevation layers:

osg::ref_ptr<osgEarth::GDALElevationLayer> layer = new osgEarth::GDALElevationLayer();
layer->setURL(path.toStdString());
map()->addLayer(layer);

When displaying the globe, it loads the tiles only very very slowly - basically not usable performance. In 2.x i was able to morph to a geolocation and instantly see the rendered globe with the geotiff imagery.

I was able to improve the performance a bit, by converting my geotiff files. However, im still not reaching the old performance, while having an increased geotiff size on disk.

gdal_translate -of GTiff -co TILED=YES input.tif output.tif

Is there anything else that i can do to improve the performance?

I have tried the solution from the following thread, but it did not seem to improve anything for the better for me.
https://github.com/gwaldron/osgearth/discussions/2049

**Comment by liu-do-program** [comment]
Hello,did you solve this problem?

**Comment by FuXiii** [comment]
I have the same problem, when I add a elevation tif file into osgearh scene, after I close to the terrain , frame rate decrease and slow. By the way I use osgEarth+Qt
